GruntFile.js

```diff
@@ -79,13 +78,14 @@ module.exports = function (grunt) {
         testname: 'atlas',
         maxRetries: 10,
         maxPollRetries: 10,
-        'public': 'public',
+        public: 'public',
```

Comment:
That surprised me a well. but it seems as though it was unnecessary. http://kangax.github.io/compat-table/es5/#Reserved_words_as_property_names and https://mathiasbynens.be/notes/javascript-properties